src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -779,26 +779,18 @@ expected::ExpectedBool VerifySign(
 }
 
 error::Error PrivateKey::SaveToPEM(const string &private_key_path) {
-	auto bio_key = unique_ptr<BIO, void (*)(BIO *)>(NULL, bio_free_func);
+	if (path::FileExists(private_key_path)) {
+		path::FileDelete(private_key_path);
```

Comment:
Unchecked error return.

Reply:
Again, there's not much to do and we don't really care if the file was actually deleted or not, all we care is if we successfully create a new file in the next step. If, for example, something deletes the file between the `FileExists()` and `FileDelete()` calls, it's not a problem for us. So perhaps a debug-level log message?